src/server/stages/repository/strapiStages.mapper.ts

```diff
@@ -0,0 +1,72 @@
+import { v4 as uuidv4 } from 'uuid';
+
+import { Domaines, OffreDeStage, OffreStageDepot, SourceDesDonnées } from '~/server/stages/domain/stages';
+import { OffreStageDepotStrapi, OffreStageResponseStrapi } from '~/server/stages/repository/strapiStages';
+
+export function mapOffreStage(response: OffreStageResponseStrapi.OffreStage): OffreDeStage {
+	return {
+		dateDeDebutMax: response.dateDeDebutMax,
+		dateDeDebutMin: response.dateDeDebutMin,
+		description: response.description,
+		domaines: response.domaines
+			?.filter((domaine) => domaine.nom !== OffreStageResponseStrapi.Domaines.Nom.NON_RENSEIGNE)
+			.map((domaine) => domaine.nom as unknown as Domaines) || [],
+		dureeEnJour: response.dureeEnJour ?? undefined,
+		dureeEnJourMax: response.dureeEnJourMax ?? undefined,
+		employeur: response.employeur ? {
+			description: response.employeur.description || undefined,
+			logoUrl: response.employeur.logoUrl || undefined,
+			nom: response.employeur.nom,
+			siteUrl: response.employeur.siteUrl || undefined,
+		} : undefined,
+		id: response.id,
+		localisation: response.localisation ? {
+			codePostal: response.localisation.codePostal || undefined,
+			departement: response.localisation.departement || undefined,
+			pays: response.localisation.pays || undefined,
+			region: response.localisation.region || undefined,
+			ville: response.localisation.ville || undefined,
+		} : undefined,
+		remunerationBase: response.remunerationBase ?? undefined,
+		slug: response.slug,
+		source: response.source || undefined,
+		teletravailPossible: response.teletravailPossible ?? undefined,
+		titre: response.titre,
+		urlDeCandidature: response.urlDeCandidature || undefined,
+	};
+}
+
+export function mapEnregistrerOffreDeStage(body: OffreStageDepot.OffreDeStageDepot): OffreStageDepotStrapi{
```

Comment:
```suggestion
export function mapToStrapiDepotOffreDeStage(body: OffreStageDepot.OffreDeStageDepot): OffreStageDepotStrapi{
```
Je trouve plus clair d'ajouter le to pour expliciter que c'est vers ça qu'on map, et StrapiDepot pour ajouter de la précision à la lecture quand cette fonction est appelée